packages/lambda-powertools-middleware-stop-infinite-loop/__tests__/index.js

```diff
@@ -0,0 +1,59 @@
+const consoleLog = jest.spyOn(global.console, 'log')
+
+const middy = require('middy')
+const CorrelationIds = require('@perform/lambda-powertools-correlation-ids')
+const stopInfiniteLoop = require('../index')
+
+const getHandler = () => middy(async () => {}).use(stopInfiniteLoop(3))
+
+const warnLogWasWritten = (f) => {
```

Comment:
```suggestion
const errorLogWasWritten = (f) => {
```

Reply:
good catch, was gonna log warn, but then it didn't make sense to be logging warning when we are throwing, and then I forgot to change the test back!